.tekton/developer-sandbox-ui-push.yaml

```
@@ -24,7 +24,7 @@ spec:
   - name: output-image
     value: quay.io/redhat-user-workloads/hcc-platex-services-tenant/developer-sandbox-ui:{{revision}}
   - name: dockerfile
-    value: ./Dockerfile
+    value: .gitmodules/Dockerfile
```

This value should be build-tools
```suggestion
    value: build-tools/Dockerfile
```